src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,187 @@
+package duke.parser;
+
+import duke.commands.AddTaskCommand;
+import duke.commands.Command;
+import duke.commands.DeleteCommand;
+import duke.commands.DoneCommand;
+import duke.commands.ExitCommand;
+import duke.commands.FindCommand;
+import duke.commands.InvalidArgumentsException;
+import duke.commands.ListCommand;
+import duke.commands.UnknownCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
```

Comment:
I like how you ordered the import statements. They are consistent. At one glance I can understand the code base easily. 👍